### DISCUSSION TITLE
DietPi-Software | Remotely

### DISCUSSION THREAD

**Discussion by mews-se** [MichaIng/DietPi#6170] [Software Request]
#### Formal software information
- Software title | Remotely
- Short description | A remote control and remote scripting solution, built with .NET 6, Blazor, and SignalR Core.
- GitHub URL (if available) | https://github.com/immense/Remotely
- (Official) install documentation | https://github.com/immense/Remotely

#### Are there similar/alternative software titles available with DietPi-Software?
- xrdp, tigervnc - but not as alternatives

#### What makes your requested software better than the above solutions, if available?
- Open Source, lots of different variants depending on OS, lot's of different security solutions

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
- Feature the software

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. Script a docker-compose.yml file to be run, it's pretty neat and simple

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- I will gladly help doing what I can!

**Comment by Joulinar** [comment]
Many thanks for your request.

I converted it into a discussion, so people can vote for it via ⬆️ button at above post.
